README.md

```diff
@@ -15,9 +15,15 @@ OpenPBR Surface is an open standard hosted by the [Academy Software Foundation](
 
 * **[White paper](https://academysoftwarefoundation.github.io/OpenPBR/)**
 * **[Parameter reference](https://academysoftwarefoundation.github.io/OpenPBR/#parameterreference)**
-* **[Reference implementation](reference/open_pbr_surface.mtlx)** – written in [MaterialX](https://materialx.org/)
+* **[MaterialX reference implementation](reference/open_pbr_surface.mtlx)**
 * **[BibTeX citation](openpbr.bib)**
 
+### Resources
+
+* [MaterialX web-viewer](https://academysoftwarefoundation.github.io/MaterialX/?file=Materials/Examples/OpenPbr/open_pbr_default.mtlx) - WebGL test renderer using MaterialX [implementation](reference/open_pbr_surface.mtlx) of OpenPBR
```

Comment:
Just a minor note, but for consistency with other documentation, I'd recommend referring to this application as "MaterialX Web Viewer".

Reply:
Done in 41271ff0a30539559b90b7de83f1640634c9952f.